portal/app/account/views.py

```diff
@@ -308,11 +308,11 @@ def unconfirmed():
     return render_template("account/unconfirmed.html")
 
 
-@account.route("/dashboard", methods=["GET", "POST"])
+@account.route("/", methods=["GET", "POST"])
 @login_required
-def dashboard():
+def index():
     """User dashboard page."""
-    return render_template("account/dashboard.html")
```

Comment:
```python
def dashboard():
    """User dashboard page."""
    if current_user.role.index == "admin":
        return render_template("account/admin-dashboard.html")
    return render_template("account/user-dashboard.html")
```

Reply:
you can then simply add these 2 html files

Reply:
There is already a function in admin/views.py to render the admin dashboard with. The views.py in that folder has all functions specific to the admin and has `@admin_required` . I believe that is the default, to have separate views.py for admin, account, and main folders.